variables.tf

```
@@ -55,6 +55,12 @@ variable "default_table_expiration_ms" {
   default     = null
 }
 
+variable "max_time_travel_hours" {
+  description = "Defines the time travel window in hours"
+  type        = number
+  default     = 168
```

Let's default to null so we can re use the provider default

```suggestion
  default     = null
```